src/Frontend.hs

```diff
@@ -172,15 +172,15 @@ type AulaMain =
 
 aulaMain :: ActionM m => ServerT AulaMain m
 aulaMain =
-       runHandler Page.viewRooms
+       forceChangeInitialPassword (runHandler Page.viewRooms)
```

Comment:
Guarded list idea spaces and not guarded delegation network pages made me feel wierd as I tested out manually:

 - I can't open idea lists page but I can see my delegation? Hmm that's wierd.

I aimed the consistency and force the user to make change his initial password.